debugging/book-library/script.js

```diff
@@ -25,79 +21,98 @@ const author = document.getElementById("author");
 const pages = document.getElementById("pages");
 const check = document.getElementById("check");
 
-//check the right input from forms and if its ok -> add the new book (object in array)
-//via Book function and start render function
+// Add event listener to the Submit button
+document.querySelector(".btn-primary").addEventListener("click", function (e) {
+  e.preventDefault();  // Prevent form submission
+  submit();  // Call the submit function
+});
+
+// Check if input is valid and then add the book
 function submit() {
+  // Validate inputs
   if (
-    title.value == null ||
-    title.value == "" ||
-    pages.value == null ||
-    pages.value == ""
+    title.value.trim() === "" ||  // Check for empty title
```

Comment:
What will happen if value is null/undefined but you are calling .trim() method?

Reply:
Thanks for your feedback, I changes my code accordingly.